services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Persistence.kt

```diff
@@ -92,17 +92,19 @@ enum class TodennakoinenHaittaPaaAjoRatojenKaistajarjestelyihin(
 }
 
 /** NOTE Järjestys täytyy olla pienimmästä suurimpaan */
-enum class KaistajarjestelynPituus(override val value: Int, override val explanation: String) :
-    Luokittelu {
-    YKSI(1, "Ei tarvita"),
-    KAKSI(2, "Enintään 10 m"),
-    KOLME(3, "11 - 100 m"),
-    NELJA(4, "101 - 500 m"),
-    VIISI(5, "Yli 500 m")
+enum class AutoliikenteenKaistavaikutustenPituus(
+    override val value: Int,
+    override val explanation: String
+) : Luokittelu {
+    YKSI(1, "Ei vaikuta kaistajärjestelyihin"),
+    KAKSI(2, "Kaistavaikutusten pituus alle 10 m"),
+    KOLME(3, "Kaistavaikutusten pituus 10-99 m"),
+    NELJA(4, "Kaistavaikutusten pituus 100-499 m"),
+    VIISI(5, "Kaistavaikutusten pituus 500 m tai enemmän")
 }
 
 /** NOTE Järjestys täytyy olla pienimmästä suurimpaan */
-enum class Haitta13 {
+enum class Haitta123 {
```

Comment:
I don't know if `Haitta123` is a lot more descriptive than `Haitta13`.

This could probably be split into three separate enums for the three use cases of noise, dust and tremors. There's not a big difference between these and say, AutoliikenteenKaistavaikutustenPituus.

Reply:
I do not understand why any numbers need to be in the name. Why not just Haitta? Also, the suggestion above is a good one I think.

Reply:
Created separate enums Meluhaitta, Polyhaitta and Tarinahaitta.